### PR TITLE
Ajc fix compression detection

### PR DESCRIPTION
**Pull request by ambrosejcarr**
- fixed sctools to detect compressed files based on magic numbers, not rely on file extensions.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 setup(
     name='sctools',
-    version='0.1.4',
+    version='0.1.6',
```
Comment (samanehsan) [inline setup.py]: Why is this version `0.1.6` and not `0.1.5`?

Comment (ambrosejcarr) [reply setup.py]: Because I need to start using the pypi test server instead of pushing things with bugs directly to pypi's main index. :)

Comment (samanehsan) [reply setup.py]: Ohh okay! What was the bug? Do you think additional tests could help or was it really minor like a typo?

Comment (ambrosejcarr) [reply setup.py]: Yea.. I think there was a testing failure. I'll think about how to add them. 👍

**Pull request merged**